rake_helpers/generate_ep_popolo.rb

```diff
@@ -100,6 +100,11 @@ def terms_from_csv
       m.delete :start_date if m[:start_date].to_s.empty? || (!e[:start_date].to_s.empty? && m[:start_date].to_s <= e[:start_date].to_s)
       m.delete :end_date   if m[:end_date].to_s.empty?   || (!e[:end_date].to_s.empty?   && m[:end_date].to_s   >= e[:end_date].to_s)
     end
+    duplicates = @json[:memberships].size - @json[:memberships].uniq.size
+    if duplicates > 0
+      warn "Discarding #{duplicates} duplicate membership#{duplicates != 1 ? 's' : ''} from Popolo".yellow
```

Comment:
This would be a lot better if if reported something about what the duplicates were, so they can actually be investigated…